stores/github/authorization-model.json

```diff
@@ -1,11 +1,26 @@
 {
   "type_definitions": [
+    {
+      "type": "user",
+      "relations": {}
+    },
     {
       "type": "team",
       "relations": {
         "member": {
           "this": {}
         }
+      },
+      "metadata": {
+        "relations": {
+          "member": {
+            "directly_related_user_types": [
+              {
+                "type": "user"
```

Comment:
should team:member be directly related to organization:member?

Reply:
We don't have that in the tuples